.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+# If this file is renamed, the incrementing run attempt number will be reset.
+
+name: CI
+
+on:
+  push:
+    branches: [ "dev", "master" ]
+  pull_request:
+    branches: [ "dev", "master" ]
+      
+env:
+  CI_BUILD_NUMBER_BASE: ${{ github.run_number }}
+  CI_TARGET_BRANCH: ${{ github.head_ref || github.ref_name }}
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 8.0.x
+      - name: Compute build number
+        run: |
+          echo "CI_BUILD_NUMBER=$(($CI_BUILD_NUMBER_BASE+200))" >> $GITHUB_ENV
```

Comment:
Just curious, why +200?

Reply:
looks like a copy/paste of https://github.com/serilog-tracing/serilog-tracing/blob/dev/.github/workflows/ci.yml

Reply:
It looks like it is just to keep this incrementing and not start at 0. 
![image](https://github.com/serilog-contrib/serilog-sinks-splunk/assets/15874076/68b19902-78e7-4974-b321-afe58c0c334f)

Reply:
Fixed, thanks